includes/resourceloader/ResourceLoaderStartUpModule.php

```diff
@@ -98,6 +98,7 @@ protected function getConfig( $context ) {
 			'wgLegalTitleChars' => Title::convertByteClassToUnicodeClass( Title::legalChars() ),
 			// Wikia - change begin - @author: wladek
 			'wgSassParams' => SassUtil::getSassSettings(),
+			'wgCommunityPageDisableTopContributors' => $wgCommunityPageDisableTopContributors,
```

Comment:
Since this is extension-specific, it should be added to the RL config vars with a hook (i.e. `ResourceLoaderGetConfigVars`).